mmpose/models/__init__.py

```diff
@@ -1,13 +1,15 @@
 from .backbones import *  # noqa
-from .builder import (BACKBONES, HEADS, LOSSES, NECKS, POSENETS,
-                      build_backbone, build_head, build_loss, build_neck,
-                      build_posenet)
+from .builder import (BACKBONES, EXTERNAL_MODELS, HEADS, LOSSES, NECKS,
```

Comment:
What does `EXTERNAL_MODELS` mean?

Reply:
Currently, it only contains SMPL. Any suggestions on the name?

Reply:
Something like `MESH_LAYERS`?

Reply:
I agree with the name `MESH_LAYERS`.
And I suggest moving the layers as well as the registry and builder into `mmpose/models/utils`.